### DISCUSSION TITLE
[IntelliSense] Suggestion not working in <script> tag in .vue file

### DISCUSSION THREAD

**Discussion by hungthezorba** [tailwindlabs/tailwindcss#12256] [Help]
Hi, i'm working a Vue project that using Tailwind and Tailwind IntelliSense. I have already config `Tailwind CSS: Class Attributes` like below
<img width="980" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/51154273/1cc6d538-a9d4-4894-90d3-1ed5ed7188ba">
However, when i declare the const `baseClasses` or an computed `baseClasses`, there is no suggestion or autocomplete for tailwind class
<img width="796" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/51154273/ebf59e04-4231-447b-8065-9e7e20fd2b3c">
<img width="799" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/51154273/3e1064ee-e78d-4ec6-95b9-00382e59f28e">

I don't know if this is not supported or it is my configuration.

**Comment by wongjn** [comment]
You could try configuring the [`tailwindCSS.experimental.classRegex` setting](https://github.com/tailwindlabs/tailwindcss/issues/7553#issuecomment-735915659).

**Reply by geminigeek** [reply]
hi you guys figured it out how to do that ?

**Reply by wongjn** [reply]
Follow the link in the above comment - it shows how to configure the setting.

**Reply by geminigeek** [reply]
hi thanks for your reply, just after posting this i found this  , easy todo from UI , i always name my variable with *Class or *Style this works for me, https://github.com/tailwindlabs/tailwindcss/discussions/7554#discussioncomment-3030988
